com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -30,9 +30,6 @@ def _matching_resource(json_rep, name=None):
         return re.match(name, json_rep['name'])
     return True
 
-  
-
-
 class _ResourceElement(object):
```

Comment:
If you are fixing whitespace, then pycharm seems to expect two blank lines before a class.